src/roles/vsc-upgrade-postdeploy/tasks/main.yml

```diff
@@ -88,13 +88,13 @@
 
 - name: Verify VSD Node1 is found in vsd_detail
   assert:
-    that: "'cna@{{ vsd_fqdn }}/{{ vsd_hostname_list[0].split('.')[0:1] | join('.') }}' in vsd_detail.stdout[0]"
+    that: "'cna@{{ vsd_fqdn }}/{{ vsd_hostname_list[0].split('.')[0:1] | join('.') }}'[:50] in vsd_detail.stdout[0]"
```

Comment:
Wasn't there a concern that checking the first 50 characters wouldn't be sufficient here?

Reply:
That will be addressed as a part of seperate PR. Did not want to tackle that as a part of current patch